cdc/sim/BUILD.bazel

```diff
@@ -0,0 +1,65 @@
+# Copyright 2024 The Bedrock-RTL Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_hdl//verilog:providers.bzl", "verilog_library")
+load("//bazel:br_verilog.bzl", "br_verilog_sim_test_suite")
+load("//bazel:verilog.bzl", "verilog_elab_test")
+
+package(default_visibility = ["//visibility:public"])
+
+verilog_library(
+    name = "br_cdc_fifo_flops_tb",
+    srcs = ["br_cdc_fifo_flops_tb.sv"],
+    deps = [
+        "//cdc/rtl:br_cdc_fifo_flops",
+        "//fifo/sim:br_fifo_test_harness",
+        "//misc/sim:br_test_driver",
+        "//pkg:br_math_pkg",
+    ],
+)
+
+verilog_elab_test(
+    name = "br_cdc_fifo_flops_tb_elab_test",
+    deps = [":br_cdc_fifo_flops_tb"],
+)
+
+br_verilog_sim_test_suite(
+    name = "br_cdc_fifo_flops_vcs_test_suite",
+    params = {
+        "Depth": [
+            # Minimum depth needed for full bandwidth at the maximum cut-through and backpressure latencies
+            "15",
```

Comment:
Is this calculation documented somewhere?

Reply:
Added it to the top-level comments